configs/pontoon.toml

```diff
@@ -128,16 +128,6 @@ locales = [
 [[paths]]
     reference = "en/firefox/nightly/**/*.ftl"
     l10n = "{locale}/firefox/nightly/**/*.ftl"
-[[paths]]
-    reference = "en/firefox/privacy/book.ftl"
-    l10n = "{locale}/firefox/privacy/book.ftl"
-    locales = [
```

Comment:
Note for the future: this is not particularly useful, because Smartling will translate this for all locales enabled.